docs/examples/getting_started.ipynb

```diff
@@ -419,7 +419,13 @@
     "\n",
     "Rotate's `angle` parameter can accept `float` or `float tensor` types of values. The second option, `float tensor`, enables us to feed the operator with different rotation angles for every image, via a tensor produced by other operation.\n",
     "\n",
-    "Random number generators are examples of operations that one can use with DALI. Let us use `fn.random.uniform` to make a pipeline that rotates images by a random angle."
+    "Random number generators are examples of operations that one can use with DALI. Let us use `fn.random.uniform` to make a pipeline that rotates images by a random angle.\n",
+    "\n",
+    "> **_NOTE_** Keep in mind, that every time you pass an oputput of one DALI operator to another as named keyword argument (like the angle for the rotate above) that data must be placed on the CPU (the random.uniform has the default device='cpu').\n",
```

Comment:
Wasn't it better to have it as a dedicated section?

Also, you can consider having a small code block with the mentioned code in this section, and mention the name of the variable that we pass as the argument input.

Reply:
I think it was easy to miss it before. Now this is in one paragraph.
Also I think the code is already described:
> (like the angle for the rotate in below example) that data must be placed on the CPU (the random.uniform has the default device='cpu').

Reply:
Ok, I agree that the placement is better, I'm not convinced about the formatting - it's not like we have warning box in notebooks, that why I was talking about dedicated section. I didn't notice how far up you moved it.

Now it references code "above", and the code is below.

Reply:
We don't, this is the closest thing I was able to come up (I'm open to suggestions). I think it deserves a highlight.

Reply:
adjusted `above->below`